src/components/views/auth/RegistrationForm.js

```diff
@@ -270,12 +271,23 @@ module.exports = React.createClass({
                     }
                     const { scorePassword } = await import('../../../utils/PasswordScorer');
                     const complexity = scorePassword(value);
+                    const unsafe = complexity.score < PASSWORD_MIN_SCORE;
```

Comment:
Let's reverse the name and meaning to reduce the amount of negation flying around:

```suggestion
                    const safe = complexity.score >= PASSWORD_MIN_SCORE;
```

Reply:
applied manually